vite.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+
+// https://vitejs.dev/config/
```

Comment:
**praise**: Including a reference to the Vite configuration documentation is helpful for future developers.